lib/capistrano/tasks/delayed_job.rake

```diff
@@ -15,6 +15,10 @@ namespace :delayed_job do
     unless fetch(:delayed_job_pools).nil?
       args << fetch(:delayed_job_pools, {}).map {|k,v| "--pool='#{k}:#{v}'" }.join(' ')
     end
+    unless fetch(:delayed_job_pools_per_machine).nil?
```

Comment:
This would obviously have to change if `:delayed_job_pools_per_machine` being used would _skip_ any settings in `:delayed_job_pools`

Reply:
Corrected version is here: 
https://github.com/goodylabs/capistrano3-delayed-job/commit/8f3b5cb768d15ddf0d58614504f1d9f25c828d0c